spread.yaml

```diff
@@ -929,7 +929,34 @@ prepare: |
             exit 1
         fi
     fi
-    
+
+    # native jq replacement, but still with some incompatibilies, see
+    # https://github.com/itchyny/gojq
+    # major differences:
+    # - map keys are sorted by default
+    # - with --yaml-input, can parse YAML
+    if os.query is-arm64; then
+        "$TESTSTOOLS"/simpleget -o /tmp/gojq.tar.gz \
+            https://github.com/itchyny/gojq/releases/download/v0.12.16/gojq_v0.12.16_linux_arm64.tar.gz
+    elif os.query is-arm; then
+        # upstream does not provide prebuilt binaries for armhf
+        # built with: GOARCH=arm GOARM=7
+        "$TESTSTOOLS"/simpleget -o /tmp/gojq.tar.gz \
+            https://storage.googleapis.com/snapd-spread-tests/dependencies/gojq_v0.12.16_linux_armhf.tar.gz
+    else
+        # must be amd64
```

Comment:
Laughs in risc

Can you make an elif and a fatal default case please?

Lastly instead of if-then-else can you switch on `uname -m`?